tasmota/xsns_69_opentherm_protocol.ino

```diff
@@ -568,7 +597,8 @@ void sns_opentherm_protocol_reset()
     for (int i = 0; i < SNS_OT_COMMANDS_COUNT; ++i)
     {
         struct OpenThermCommandT *cmd = &sns_opentherm_commands[i];
-        cmd->m_flags.m_flags = 0;
+        cmd->m_flags.notSupported = cmd->m_flags.manual;
```

Comment:
I know you covered all use cases here, however, if someone else will add another flag in the future and forget to change the logic here, he might end up with a buggy code.
I would suggest doing something like:
```
bool isManual = cmd->m_flags.manual;
cmd->m_flags.m_flags = 0;
if (isManual) {
  cmd->m_flags.manual = 1;
  cmd->m_flags.notSupported = 1;
}
```

Reply:
cmd->m_flags.m_flags = 0;

will reset supported flag to 0. I think it's a bug